### ISSUE TITLE
JSON error with app.js

### ISSUE THREAD

**Issue by hswick** [TrueBitFoundation/webasm-solidity#20]
Was running through the README and was able to run deploy-tasks.js with parity chain dev to create the config.json. But when I try to run app.js it gives a JSON error.

This is because deploy-tasks.js is outputting two json objects, instead of one, like app.js expects.
```
{ "interactive": "0x4d06047ed426e2a2f0bbe86cee268200d20b8d62", "tasks" : "0xcfa55b9b5c1c05e81a7f306a841ae6f3b489d057" }
{ "interactive": "0x7a0d30234e146a88cd65f0714ab7603b0568b16a", "tasks" : "0x6ec4905ff2bfbb1ee8f4c11006ef528df4d30b80" }
```
I can go in the file and delete the second object and then the app works.

**Comment by hswick** [comment]
Tried it again, and now it is only outputting 1. Not sure what I did differently. I'm gonna blame myself, and close this.

**Comment by mrsmkl** [comment]
Might be a problem with web3.js. Added a fix in #22
